com.unity.cinemachine/Editor/Utility/CinemachineEditorAnalytics.cs

```diff
@@ -11,13 +14,38 @@ static class CinemachineEditorAnalytics
         const int k_MaxEventsPerHour = 360;
         const int k_MaxNumberOfElements = 1000;
         const string k_VendorKey = "unity.cinemachine";
-        
+        const string k_CreateVcamEventName = "cm_create_vcam";
+        const string k_VcamsOnPlayEventName = "cm_vcams_on_play";
+
         // register an event handler when the class is initialized
         static CinemachineEditorAnalytics()
         {
             EditorApplication.playModeStateChanged += SendAnalyticsOnPlayEnter;
         }
 
+#if UNITY_2023_2_OR_NEWER
+        [AnalyticInfo(eventName: k_CreateVcamEventName, vendorKey: k_VendorKey, 
```

Comment:
I don't see it when creating a VCam from add component is it expected?

Reply:
It never worked from AddComponent.  It's only invoked from the entries in the Cinemachine menu.  Does it work correctly when using the menu?

Reply:
Yes!